lib/pgp-secret.ts

```diff
@@ -91,17 +104,21 @@ export class PGPSecret extends cdk.Construct {
     const secret = new cfn.CustomResource(this, 'Resource', {
       lambdaProvider: fn,
       properties: {
+        resourceVersion: hashFileOrDirectory(codeLocation),
```

Comment:
raise an issue to add this as a feature to CustomResource

Reply:
awslabs/aws-cdk#1401